### DISCUSSION TITLE
Segmind Stable Diffusion Model (SSD-1B)

### DISCUSSION THREAD

**Discussion by barepixels** [lllyasviel/Fooocus#809] [General]
The Segmind Stable Diffusion Model (SSD-1B) is a distilled 50% smaller version of the Stable Diffusion XL (SDXL), offering a 60% speedup while maintaining high-quality text-to-image generation capabilities. It has been trained on diverse datasets, including Grit and Midjourney scrape data, to enhance its ability to create a wide range of visual content based on textual prompts.

This model employs a knowledge distillation strategy, where it leverages the teachings of several expert models in succession, including SDXL, ZavyChromaXL, and JuggernautXL, to combine their strengths and produce impressive visual outputs.

Special thanks to the HF team  especially [Sayak](https://huggingface.co/sayakpaul), [Patrick](https://github.com/patrickvonplaten) and [Poli](https://huggingface.co/multimodalart) for their collaboration and guidance on this work.

I tested with Fooocus  2.1.756, MRE, RuinedFooocus.  It works

Link to model:
https://huggingface.co/segmind/SSD-1B

Reddit:
https://www.reddit.com/r/StableDiffusion/comments/17ig80u/comfy_ui_now_supports_ssd1b_upto_70_speed_up_on/?sort=old

**Comment by aimarxjg** [comment]
K佬到处溜达~
